Thor Method.py

```diff
@@ -0,0 +1,224 @@
+# 
+# Looking Back
+# ^^^^^^^^^^^^^^^^^
+# Looking back, if I was going to do the game in C++, I really should have done the codechat in C++ also. I am learning Python now to make this a little easier on me so I don't have to switch back and forth. I would copy and paste the C++ code I used in the game; however, python doesn't care for C++ and keeps putting errors at the top. 
+#
+# Soultion: Second Try with Python
+# ----------------------------------------
+# With this second try, I am going to work in `Python <https://wiki.python.org/moin/BeginnersGuide>`_.
+#
+# My Thoughts and Process
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# The whole idea is to get Thor to a power source using the xy plane. So if Thor is at (0,3) and the light is at (1,3) then Thor needs to move to the right one. Since postive x is to the right (**East**) and postive y is down (**South**). Another given is that x is 0-40 and y is 0-18 and the coodinates start at the top left of the boundry. 
+#
+# However, as I found out this my first try, Thor's position isn't being tracked, just his intial position. This needs to taken into account. We already found out with try 1 that loops aren't the best way to complete this code. Maybe we can use math instead. 
+#
+#  Example: 
+#
+#   If Thor is at (0,0) and the light is at (40,0). We know just by looking at this that Thor is -40 from the light in the x direction. So to solve this we need to add 40 to Thor's intial position.
+#
+print(initial_tx, initial_ty, light_x, light_y)  
+#
+#| The above code gets the program to show us where Thor and the light start.
+#
+x_steps = [int]  # this is going to track our difference in Thor's x and Light's x.
+y_steps = [int]  # this is going to track our difference in Thor's y and Light's y.
+
+x_steps = initial_tx - light_x  # examples:  -40 = 0 - 40 (E); 40 = 40 - 0 (W)
+y_steps = initial_ty - light_y  # examples:  -18 = 0 - 18 (S) ; 18 = 18 -0 (N)
+
+if x_steps == 0:
+    if y_steps > 0:
+        print("N")
+        y_steps -= 1
+    if y_steps < 0:
+        print ("S") 
+        y_step += 1
+elif y_steps == 0:
+    if x_steps > 0:
+        print("W")
+        x_steps -= 1
+    if x_steps < 0:
+        print("E")
+        x_steps += 1
+
+# So I spent almost an hours tryin to get that version to work however it kept giving me a time out error. So instead of working with new varibles. Lets try working with the ones we already have.
+#
+# Let's try increment Thor's initial position with every move. 
+#
+# game loop
+while True:
+    remaining_turns = int(input())  # The remaining amount of turns Thor can move. Do not remove this line.
+
+    # Write an action using print
+    # To debug: print("Debug messages...", file=sys.stderr, flush=True)
+
+    #
+    #| The above code gets the program to show us where Thor and the light start.
```

Comment:
The use of `#| Comments here` produces a monospaced line -- why do this?

Reply:
Think that was part of the given code.